benchmark.py

```diff
@@ -55,56 +56,66 @@ def _exec_command(args, shell=False, fail_if_nonzero=True):
       args, shell=shell, stdout=fd_devnull, stderr=fd_devnull)
 
 
+def _get_commits(commits_list, repo, flag_name):
+  """Returns a list of commits.
+
+  If the input commits_list is empty, fetch the latest commit from repo.
```

Comment:
really the latest commit or the latest in `master`?

Reply:
Fixed.
As discussed offline, I always use "Squash and merge" for PRs in this repository. Disabled the "Merge commits" option in the repository settings to be sure.